test/v1alpha1/route.go

```diff
@@ -87,7 +87,8 @@ func RetryingRouteInconsistency(innerCheck spoof.ResponseChecker) spoof.Response
 // interval until inState returns `true` indicating it is done, returns an
 // error or timeout. desc will be used to name the metric that is emitted to
 // track how long it took for name to get into the state checked by inState.
-func WaitForRouteState(client *test.ServingAlphaClients, name string, inState func(r *v1alpha1.Route) (bool, error), desc string) error {
+// ignoreNotFound indicates that we will keep polling even if the route is not found.
+func WaitForRouteState(client *test.ServingAlphaClients, name string, inState func(r *v1alpha1.Route) (bool, error), desc string, ignoreNotFound bool) error {
```

Comment:
Lets avoid updating this signature. See comment above for what I think is going on here.